### PR TITLE
Telseq 0.0.1 (new formula)

### PR DESCRIPTION
**Pull request by danielecook**

### CODE REVIEW

File: telseq.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Telseq < Formula
+  desc "Program for estimating telomere length from sequence data"
+  homepage "https://github.com/zd1/telseq"
+  # doi "10.1093/nar/gku181"
+  # tag "bioinformatics"
+
+  url "https://github.com/zd1/telseq/archive/0.0.1.tar.gz"
+  sha1 "b502b6695af2ac87c6ad28e77113866b63991441"
```
Comment (dpo) [inline telseq.rb]: Please use the `sha256`.

File: telseq.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  head "https://github.com/zd1/telseq.git"
+
+  depends_on "gcc"
```
Comment (dpo) [inline telseq.rb]: There's no need to depend on `gcc` but if this doesn't build with `clang`, then there should be a `fails_with` block.

**Comment by jonchang** [comment]
ping @danielecook

**Comment by iMichka** [comment]
Closing here as there was no activity since some time now. If somebody really needs this a new PR can be opened.

**Pull request closed without merging**